### ISSUE TITLE
for Subscript events, don't emit a column 'highlight' entry if you're already emitting a column-to-column outline (i.e., arrow)

### ISSUE THREAD

**Issue by pgbovine** [SamLau95/pandas_tutor#51]
In examples like these where you're subscripting to pull out an entire column:

```python
import pandas as pd
import io
csv = """
breed,grooming,food_cost,kids,size
Labrador Retriever,weekly,466.0,high,medium
German Shepherd,daily,466.0,medium,large
"""
dogs = pd.read_csv(io.StringIO(csv))
# this should emit only an arrow and *not* a highlight on grooming column
dogs["grooming"]
```

The trace is:
```
      "mapping": [
        {
          "select": "column",
          "anchor": "lhs",
          "label": "grooming",
          "illustrate": "highlight"
        },
        {
          "select": "column",
          "from": {
            "anchor": "lhs",
            "label": "grooming"
          },
          "to": {
            "anchor": "rhs",
            "label": "pandas.Series"
          },
          "illustrate": "outline"
        }
      ],
```

I'd like to get rid of this highlight:
```
        {
          "select": "column",
          "anchor": "lhs",
          "label": "grooming",
          "illustrate": "highlight"
        },
```

The 'outline' (which my script turns into an arrow) already contains this info, and my canonicalization script automatically puts the box around lhs. These highlights in the trace are making my script output pretty confusing, and it also conflates another way that 'highlight' is being used (which i'd like to keep).

The other way that highlight seems to be being used is as filter conditions for columns, which i want to keep.

In this related example:
```python
import pandas as pd
import io
csv = """
breed,grooming,food_cost,kids,size
Labrador Retriever,weekly,466.0,high,medium
German Shepherd,daily,466.0,medium,large
"""
dogs = pd.read_csv(io.StringIO(csv))
# this should emit a highlight in the grooming column since it is a filter condition
dogs.loc[dogs["grooming"] == "daily"]
```

The trace is:
```
      "mapping": [
        {
          "select": "column",
          "anchor": "lhs",
          "label": "grooming",
          "illustrate": "highlight"
        },
        {
          "select": "row",
          "from": {
            "anchor": "lhs",
            "label": 1
          },
          "to": {
            "anchor": "rhs",
            "label": 1
          },
          "illustrate": "outline"
        }
      ],
```

Note that there's only one column select highlight here, and that indicates the filter condition (we're filtering on 'grooming'). The 'outline' (arrow) refers to rows, so that doesn't interfere. This is an example of a highlight that i definitely want to KEEP. It's the same with SortValuesCall highlight column too, as in ...

```python
import pandas as pd
import io
csv = """
breed,grooming,food_cost,kids,size
Labrador Retriever,weekly,466.0,high,medium
German Shepherd,daily,466.0,medium,large
"""
dogs = pd.read_csv(io.StringIO(csv))
# this should emit a highlight in the grooming column since it is a sort condition
dogs.sort_values("grooming")
```

```
    "mapping": [
        {
          "select": "column",
          "anchor": "rhs",
          "label": "grooming",
          "illustrate": "highlight"
        },
        {
          "select": "row",
          "from": {
            "anchor": "lhs",
            "label": 0
          },
          "to": {
            "anchor": "rhs",
            "label": 0
          },
          "illustrate": "outline"
        },
        {
          "select": "row",
          "from": {
            "anchor": "lhs",
            "label": 1
          },
          "to": {
            "anchor": "rhs",
            "label": 1
          },
          "illustrate": "outline"
        }
      ],
```
(which i'd like to keep)

The bottom line is that 'highlight' column should semantically mean filter/sorting condition, since that's how my canonicalization/frontend stuff is treating it.

**Comment by pgbovine** [comment]
update: i see that the AssignCall tests in your test suite also highlight columns, but i think that's fine since they're not of type "Subscript" ... they're a different type, so i can tell them apart. i think what i was having trouble with is the fact that Subscript means so many different things, so it seems to me that a column highlight in "Subscript" should mean the filtering/sorting condition.
